### PR TITLE
Clean up initial fullscreen and update border when title changes

### PR DESCRIPTION
**Pull request by okratitan**
This fixes the "flicker" that was seen when windows were initially set to fullscreen where the window would show up for a second and then go fullscreen.

This also now handles the property change notify hint so that we can start updating window properties as they change.  The first step is to update the window title when the name changes.

### CODE REVIEW

File: wm/frame.go [hunk 1]
```diff
@@ -356,6 +359,11 @@ func (f *frame) applyTheme(force bool) {
 	}
 }
 
+func (f *frame) setTitle() {
```
Comment (andydotxyz) [inline wm/frame.go]: A setter should take a parameter - this probably needs a different name?

File: wm/frame.go [hunk 2]
```diff
@@ -356,6 +359,11 @@ func (f *frame) applyTheme(force bool) {
...
+	//Should we somehow just be changing the label rather than forcing a theme apply?
```
Comment (andydotxyz) [inline wm/frame.go]: This is not really a code document - more like a TODO or question, does it belong here?
If this is not the right way to do something should it be pushed?

p.s. Go likes a space after the slashes ;)

File: wm/frame.go (hunk 1 above)
Comment (okratitan) [reply wm/frame.go]: Fixed.

File: wm/frame.go (hunk 2 above)
Comment (okratitan) [reply wm/frame.go]: Removed the comment - After looking at the code is proper because applyTheme() handles refresh rather completely redoing the theme.

**Pull request merged**